### PR TITLE
Release preparation

### PR DESCRIPTION
**Pull request by wtritch**
Prepare for releasing SS_SharedMultiJar as version 3.0.0
This will be our first release of a legacy project as versioned artifacts into our repository.
The SS_SharedMultiJar's root POM functions as both the parent pom for the shared sub-module artifacts AND as a shared parent POM for many other SBAC projects.

I had to break the circular dependency between SS_SharedMultiJar and TDS_TestDeliverySystemDataAccess by removing the TDS_Test versions from the dependencyManagement section of the parent POM.  This will require moving TDS_Test version declarations into the projects into the projects that actually have dependencies.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	<groupId>org.opentestsystem.shared</groupId>
 	<artifactId>shared-master</artifactId>
 	<name>Master POM for open test system shared modules</name>
-	<version>1.0.1-BUILD-SNAPSHOT</version>
+	<version>3.0.0-SNAPSHOT</version>
```
Comment (wtritch) [inline pom.xml]: We will be releasing the parent POM and all submodules as a single-version set of artifacts.  (This means that if something changes in 'shared-db,' for example, we will need to re-release the entire set and bump the parent pom version on any child projects.)

File: pom.xml [hunk 2]
```diff
@@ -21,11 +21,6 @@
 		<log4j.xml.AppLogLevel>info</log4j.xml.AppLogLevel>
 		<log4j.xml.RootLogLevel>info</log4j.xml.RootLogLevel>
 
-
-		<!--NOTE: this should be set to the non snapshot version i.e. if this pom's 
-			version is 1.0-SNAPSHOT, the property should be set to 1.0 -->
-		<shared-quality-config.version>0.0.4-SNAPSHOT</shared-quality-config.version>
```
Comment (wtritch) [inline pom.xml:27]: This was unused.

File: pom.xml [hunk 3]
```diff
@@ -108,11 +103,6 @@
 		<slf4j.version>1.7.2</slf4j.version>
 		<xmlunit.version>1.5</xmlunit.version>
 		<woodstox.version>4.4.0</woodstox.version>
-
-		<!-- Default versions for TDS projects -->
-		<tds-shared.version>1.0.1-BUILD-SNAPSHOT</tds-shared.version>
-		<tds-dll.version>0.0.1-SNAPSHOT</tds-dll.version>
```
Comment (wtritch) [inline pom.xml:114]: Removed the tds-dll (TDS_TestDeliverySystemDataAccess) dependency from this project.
Replaced the tds-shared.version property with references to \${project.version}

File: pom.xml [hunk 4]
```diff
@@ -823,5 +780,20 @@
...
+		<tag>HEAD</tag>
+	</scm>
+
+	<distributionManagement>
+		<repository>
+			<id>central</id>
+			<name>airdev-releases</name>
+			<url>https://airdev.jfrog.io/airdev/libs-releases-local</url>
+		</repository>
+	</distributionManagement>
```
Comment (wtritch) [inline pom.xml:849]: Added release deployment information.

File: shared-mq/pom.xml [hunk 5]
```diff
@@ -1,41 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (wtritch) [inline shared-mq/pom.xml:2]: The shared-mq submodule wasn't referenced as a submodule from the parent pom, and there were no dependencies on it in source code.  I removed it for cleanliness.

**Review by malaffoon — APPROVED** [review]

**Comment by wtritch** [comment]
I'm going to hold off on merging back into 'develop' until after we've released version 3.0.0 and changed all of the other projects to depend on version 3.0.0 rather than 1.0.1-BUILD-SNAPSHOT

**Comment by wtritch** [comment]
I've updated all of the dependent projects to reference the released version 3.0.1.  I'm now merging this branch into 'develop'

**Pull request merged**